target/linux/generic/hack-6.12/931-fix-compilation-warnings.patch

```diff
@@ -0,0 +1,34 @@
+--- a/include/linux/irq.h
++++ b/include/linux/irq.h
+@@ -1267,6 +1267,9 @@ static inline u32 irq_reg_readl(struct i
+ 		return readl(gc->reg_base + reg_offset);
+ }
+ 
++int get_c0_perfcount_int(void);
++unsigned int get_c0_compare_int(void);
```

Comment:
> generic: 6.12: fix compilation warnings

Please drop this commit from the patchset. We can handle the MIPS target later.

Reply:
Ok.